includes/tracks/events/class-wc-admin-setup-wizard-tracking.php

```diff
@@ -19,9 +19,25 @@ public static function init() {
 			return;
 		}
 
+		add_action( 'add_option_woocommerce_allow_tracking', array( __CLASS__, 'track_start' ), 10, 2 );
 		self::add_step_save_events();
 	}
 
+	/**
+	 * Track when tracking is opted into and OBW has started.
+	 *
+	 * @param string $option Option name.
+	 * @param string $value  Option value.
+	 * @return void
+	 */
+	public static function track_start( $option, $value ) {
```

Comment:
I can't find documentation on this action. Will `$option` always be the correct option name? I assume so, but figured I'd ask.

Reply:
Yep, `$option` will always be correct.  Seems a bit redundant, but there's probably a use case for a generic function that handles a bunch of options.